kolibri/plugins/setup_wizard/assets/src/views/onboarding-forms/SettingUpKolibri.vue

```diff
@@ -194,6 +195,9 @@
         return this.wizardService.state.context[key];
       },
       provisionDevice() {
+        // Restart ZeroConf (needs to be done before provisioning)
+        FinishSoUDSyncingResource.postListEndpoint('restart');
```

Comment:
As this is a non-blocking async call, this is going to cause a weird race condition, and as @bjester pointed out, this should be happening _after_ provisioning has occurred, so that we restart zeroconf when the `subset_of_users_device` flag has been updated.

Reply:
Bah I didn't remember off hand when I read your comment, but I did that initially but if I try that API endpoint after provisioning I'd get 403 because those SetupWizard-specific permissions classes rely on the device not yet being provisioned.

So I just did the thing in the view after the serializer validation has passed and we've logged the user in before returning from the device provisioning view.

Reply:
That also seems fine to me!